packages/stg-devtools-evm-hardhat-v2/src/asset/sdk.ts

```diff
@@ -47,7 +55,12 @@ export class Asset extends Ownable implements IAsset {
 
         return {
             ...this.createTransaction(data),
-            description: `Setting address config for ${formatOmniPoint(omniContractToPoint(this.contract))}:\n${printRecord(config)}`,
+            description: `Setting address config for ${formatOmniPoint(omniContractToPoint(this.contract))}`,
+            metadata: {
+                contractName: this.contractName,
+                functionName: 'setAddressConfig',
+                functionArgs: `config = ${printRecord(config)}`,
+            },
```

Comment:
Currently not printing the new fields, even after cleaning cache, installing dependencies, and building. Will continue investigating!

<img width="778" alt="Screenshot 2024-11-19 at 3 50 04 PM" src="https://github.com/user-attachments/assets/8f1dd15f-c5f8-4df7-a0df-6a3ababb2761">

Reply:
But appears in devtools when wiring:

<img width="1382" alt="Screenshot 2024-11-20 at 10 59 20 AM" src="https://github.com/user-attachments/assets/46e9dabd-a821-440e-9bb3-7973ca4ed6b0">

Reply:
waitamin

network is the difference here...perhaps I need to update the formatting elsewhere

Reply:
ok i think stg actually uses the [devtools-evm-hardhat version of fomatOmniTransaction](https://github.com/LayerZero-Labs/devtools/blob/447af656cd2e6fdfb4890fc3a4995d2089fd045a/packages/devtools-evm-hardhat/src/transactions/format.ts#L8)

so I likely need to update devtools-evm-hardhat's _devtools_ dependency to be the latest

Reply:
The reason the wire task showed the updated logs is because it imports `formatOmniTransaction` from `devtools` instead of from `devtools-evm-hardhat`

Reply:
hmm wait but we set the devtools dependency for devtools-evm-hardhat as `0.4.0` so it should still pick up `0.4.1` since it is just a patch change...

Reply:
Hmm updating `devtools-evm-hardhat` to have `devtools` version with the updated logs did not seem to fix this. The new metadata fields don't seem to print in the stargate repo but they print in the devtools repo. 

I made sure to delete all artifacts/cache/any other build files and ran `pnpm i && pnpm build` before trying to log an omnitransaction:
<img width="1535" alt="Screenshot 2024-11-21 at 11 50 05 AM" src="https://github.com/user-attachments/assets/221248d5-9c84-4e4e-9d26-09a3332ed810">